contracts/lib/NonMerklizedIssuerBase.sol

```diff
@@ -18,4 +19,13 @@ abstract contract NonMerklizedIssuerBase is INonMerklizedIssuer {
     function getCredentialProtocolVersion() external pure returns (string memory) {
         return CREDENTIAL_PROTOCOL_VERSION;
     }
+
+    /**
+     * @dev supportsInterface. Check if the contract supports the interface
+     */
+    function supportsInterface(bytes4 interfaceId) public view override returns (bool) {
```

Comment:
I think it shouldn't be in the abstract class, because it may conflict in the child class with own or other parent implementations.

Reply:
I added `virtual` instead of `abstract`